docs/metrics.md

```diff
@@ -327,6 +327,21 @@ A gauge of the number of queue items that have not been processed yet.
 
 See [queue adds count](#queue_adds_count) for details.
 
+#### `version`
+
+Build metadata for this Controller.
+
+| attribute        | explanation                                                                                           |
+|------------------|-------------------------------------------------------------------------------------------------------|
+| `version`        | The version of Argo                                                                                   |
+| `platform`       | The [Go platform](https://go.dev/doc/install/source#environment) compiled for. Example: `linux/amd64` |
+| `goversion`      | Version of Go used                                                                                    |
```

Comment:
```suggestion
| `go_version`     | Version of Go used                                                                                    |
```

same as above and https://github.com/argoproj/argo-workflows/pull/13269#discussion_r1661378969